graphsync.go

```diff
@@ -94,11 +94,12 @@ const (
 	RequestCancelled = ResponseStatusCode(35)
 )
 
-// RequestContextCancelledErr is an error message received on the error channel when the request context given by the user is cancelled/times out
-type RequestContextCancelledErr struct{}
+// RequestClientCancelledErr is an error message received on the error channel when the request is cancelled on by the client code,
+// either by closing the passed request context or calling CancelRequest
+type RequestClientCancelledErr struct{}
 
-func (e RequestContextCancelledErr) Error() string {
-	return "Request Context Cancelled"
+func (e RequestClientCancelledErr) Error() string {
+	return "Request Cancelled By Client"
```

Comment:
non-blocker/thinking out loud: I think `staticcheck` would find this [sacrilegious](https://staticcheck.io/docs/checks#ST1005) when we come to integrate unified CI stuff in this repo. 

Please feel free to ignore this comment as I can see error message capitalisation is almost consistently used across the repo.

Reply:
damn yea, probably. I never much cared for that rule, but I guess we'll fix it when we get there!